src/services/repo/useRepoOverview.tsx

```diff
@@ -122,6 +122,7 @@ export function useRepoOverview({
         }
 
         const coverageEnabled = data.owner.repository.coverageEnabled ?? false
+        const isPrivate = data.owner.repository.private ?? false
```

Comment:
hmmmm should we default this to true just in case? Or is this just for the lock icon